benchmarks/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -349,12 +361,23 @@ int main_(std::vector<std::string> const &args, const MPI_Comm comm)
     }
     }
 
-    for (int i = 0; i < n; ++i)
-      random_points_host(i) = {
-          {offset_x + random(), offset_y + random(), offset_z + random()}};
+    for (int i = 0; i < n_values; ++i)
+      random_points_host(i) = {{a_values * (offset_x + random()),
+                                a_values * (offset_y + random()),
+                                a_values * (offset_z + random())}};
+    int const max_offset = 2 * shift * i_max;
+    for (int i = 0; i < n_queries; ++i)
+      random_queries_host(i) = {
+          {a_queries * ((offset_x + random()) / 3 + max_offset / 3),
```

Comment:
I *think* this is correct. But could you please add a comment explaining what you do here, and what are the sizes and placement of queries?